tests/conftest.py

```diff
@@ -33,17 +33,17 @@ def __init__(self, testbed_file):
             topo = csv.DictReader(f)
             for line in topo:
                 tb_prop = {}
-                name = ''
+                name = ""
```

Comment:
To have all quotes in the current file in one style.

Reply:
IMO, we can't control any other guy who decides to use single quotes unless there is a code convention document and code convention checks tools integrated in CI
On the other hand such changes bloat the commit, introducing not related diff in history, e.g. person looking for last author of the line via 'git blame' will see you however there is no change in logic, code structure or functionality made by you.
I would suggest creating another PR dedicated for such cosmetic change so that others who browse the history of the file can effectively skip them, however that means you need to convince community to use this style and not the other which is not possible without agreed code conventions

Reply:
"I would suggest creating another PR dedicated for such cosmetic change so that others who browse the history of the file can effectively skip them" - agree.
"however that means you need to convince community to use this style and not the other which is not possible without agreed code conventions" - project should have agreed style convention.

So lets leave this as it is?

Reply:
"So lets leave this as it is?" contradicts with you agreement on another PR, my preference is another PR that fixes this in all pytest tests.
" project should have agreed style convention" - ideally yes, sonic doesn't

Reply:
Ok. This minor updates will be removed. Thank you.